doc/migration/README.md

```diff
@@ -0,0 +1,34 @@
+# Legacy CMS database
+
+This directory contains documentation and tooling for working with exports of data from the legacy OITE website database.
+
+Legacy data is stored in a Mysql database. 
+The existing vendor can provide exports of the data in typical `mysqldump` format (.sql.gz files in a .tar.gz wrapper).
+
+The database is large, containing _thousands_ of tables. This makes it challenging to work with.
+
+## Exporting data
+
+This directory includes a script, `export-data.sh`, that can be used to export relevant data from a database dump.
+
+To run this script, you'll need:
+
+- A POSIX-compatible shell (tested only on macOS so far)
+- Docker
+
+To run the exporter:
+
+1. Obtain a database dump in `.tar.gz` format.
```

Comment:
[suggestion (nonblocking)] should we add in that it should be obtained from OITE, and not the old vendor directly?